### ISSUE TITLE
Tooltip in not passing accessibility requirements

### ISSUE THREAD

**Issue by pavel-shermayster-sage** [ag-grid/ag-grid#6237]
```
[x] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[] support request => see 'Requesting Community Support'
```

Ag-grid tooltip disappears when you hover over its content. According to [WCAG 1.4.13](https://www.w3.org/WAI/WCAG21/quickref/#content-on-hover-or-focus) it should be possible. 
[https://www.w3.org/WAI/WCAG21/Techniques/failures/F95.html](https://www.w3.org/WAI/WCAG21/Techniques/failures/F95.html)

AG-grid version: 29.1.0

**Comment by kiril-matev** [comment]
Hello @pavel-shermayster-sage,

Thank you for bringing this up with us. 

The functionality you described wasn't part of the requirements when we built the tooltips feature. This is why we have added a feature request in our backlog to track this requirement and implement it in the future.

You can track it with this reference:
**AG-3523 [Tooltip] Allow custom tooltips to remain visible as long as the mouse hovers the tooltip (even if it leaves the bounds of the cell the tooltip is for) to allow interacting with tooltip content and for accessibility support**

You can follow the status of feature requests, bugs and releases using our pipeline:
https://www.ag-grid.com/ag-grid-pipeline/

**Now that this requirement is recorded in our backlog, we will soon close the GitHub issue because we will not update it when this feature is added in a future version of AG Grid. The best way to get an update on this item is to sign up for AG Grid new release notifications. Once a new AG Grid version is released, look for the item reference above (AG-XXXX) on the changelog page to see if it's resolved in that version:
https://www.ag-grid.com/ag-grid-changelog/**

Thanks again for bringing this up with us - we appreciate your time!

**Comment by pavel-shermayster-sage** [comment]
Thank you for the response
